# 高地

# 2号环形高地占领状态
bool is2Occupied
# 3号梯形高地占领状态
bool is3Occupied
# 4号梯形高地占领状态
bool is4Occupied